db/common/src/main/java/org/commonjava/indy/db/common/AbstractStoreDataManager.java

```diff
@@ -272,6 +288,40 @@ protected boolean store( final ArtifactStore store, final ChangeSummary summary,
 
         final StoreKey storeKey = store.getKey();
 
+        logger.warn("Storing {} using operation lock: {}", store, opLocks);
+
+        if(configuration != null) {
+            ArtifactStoreValidateData validateData = null ;
+            try {
+
+                logger.warn("=> [AbstractStoreDataManager] Check Config: " + configuration );
+
+                if(configuration.isSSLRequired() && sslStoreValidationConfig.getSslValidation() ) {
```

Comment:
I think if the internal validation config point is set to true, then we should validate. It shouldn't hinge on whether ssl is required...that's something the validator itself should look at, and validate accordingly.